### ISSUE TITLE
Add the possibility to activate/deactive features in front page dashboard for customers

### ISSUE THREAD

**Issue by pwallner** [georgestephanis/two-factor#189]
I have a website with woocommerce and I block the access to the dashboard to my customers. Would it be possible to add 2FA to the front page dashboard, so my customers can activate/deactivate/delete at least google authenticator in the front page. Nice would be if they have all options (email, google, U2F, one time passwords incl. regenerate)

**Comment by monecchi** [comment]
Good point @pwallner! The plugin should provide a few action/filter hooks in order to do that.

**Comment by pgreivel** [comment]
That sounds like a great idea. I ran into the same issue. So would love to have this possibility too! :)

**Comment by phoolto** [comment]
Would be a good idea to do this through a shortcode so you can add it to you desired location. For example add it to the WC menu.

**Comment by ecgroupcom** [comment]
Add the possibility of activating / deactivating features in the main page panel for clients either in Woocommerce, BBpress, BuddyPress, WC Marketplace ... that the client activates this security in the same way, personalize each message sent if This has activated verification by mail.

**Comment by DestructiveBurn** [comment]
O, dear, this was mentioned on Aug 4, 2017, and it still hasn't been implemented into the Two Factor.
This really should get done for those who block the backend dashboard and use the WooCommerce login system.   There are other two-step plugins doing it.

It's time to add it!

**Comment by NicolasKulka** [comment]
+1 pour l'ajout front Woocommerce

**Comment by AMorgaut** [comment]
I also want to allow Two factor connection for my woocomerce clients

It would mean either that:
- when the user goes to its "my information" page accessible from "my account", from where he can renew his password, he would find the Two factor settings option
- or better, an addidionnatal "two factor authentication' menu item would be proposed after the "my informations" one, and serve a dedication setting menu
